Source/WTF/wtf/Assertions.h

```diff
@@ -765,9 +770,19 @@ constexpr bool assertionFailureDueToUnreachableCode = false;
 
    The number of CONJECTURE_ASSERTs should not be growing unboundedly, and they should not
    stay in the codebase perpetually.
- */
+
+   Note: there is no EWS coverage for CONJECTURE_ASSERTs. So, if you add one, you are
```

Comment:
Please consider removing the "note" prefix. Every comment is a note.

I also have a nitpick for the comment above - "If the conjecture is proven false it" seems to have some words left in accidentally while editing.

Reply:
I normally use "Note:" to call extra attention to specific points.  However, in this case, after re-reading the comment in entirety, there's no reason that this one detail should stand out over the others.  So, I removed it ... and fix the other typo.